felix/fv/infrastructure/topology.go

```diff
@@ -289,6 +289,8 @@ func StartNNodeTopology(n int, opts TopologyOptions, infra DatastoreInfra) (tc T
 		// host.  So, disable CTLB handling for subsequent Felixes.
 		if i > 0 {
 			optsPerFelix[i].ExtraEnvVars["FELIX_BPFConnectTimeLoadBalancingEnabled"] = "false"
+			optsPerFelix[i].ExtraEnvVars["FELIX_BPFConnectTimeLoadBalancing"] = string(api.BPFConnectTimeLBDisabled)
+			optsPerFelix[i].ExtraEnvVars["FELIX_BPFHostNetworkedNATWithoutCTLB"] = string(api.BPFHostNetworkedNATEnabled)
```

Comment:
I think this should be also disabled. We do not want to install the workaround. CTLB is installed test-wide. We do not want to install it multiple times. That is why the option `FELIX_BPFConnectTimeLoadBalancing=false` is overriden above. But this one should be kept as set by the test.